src/onboarding/createFolder.js

```diff
@@ -1,87 +1,88 @@
+function createFolder(folderName, ownerEmail) {
+  var parentFolderId = PropertiesService.getScriptProperties().getProperty("PERSONAL_PARENT_FOLDER_ID");
+  var personalPlanTemplateId = PropertiesService.getScriptProperties().getProperty("PERSONAL_PLAN_TEMPLATE_DOC_ID");
+  
+  var parentFolder = DriveApp.getFolderById(parentFolderId);
+  var subFolders = parentFolder.getFolders();
+  var userFolder;
+  var createdDate;
+    
+  var existFolders = parentFolder.searchFolders('title = "' + folderName + '"');
+  if (!existFolders.hasNext()) {
+    userFolder = parentFolder.createFolder(folderName);
+    var userFolderId = userFolder.getId();
+
+    var personalPlanTemplateDoc = DriveApp.getFileById(personalPlanTemplateId);
+    var makeCopy = personalPlanTemplateDoc.makeCopy(folderName + ' Coderbunker Resident Freelancer', userFolder);
+    
+    createdDate = new Date();
+    
+    try {
+      userFolder.setOwner(ownerEmail);  // TODO: cannot set a different domain user as owner!
+    } catch (err) {
+      logInfo2StackdriverLogging("[Onboarding - create folder] " + err);
```

Comment:
rethrow and catch at top-level. display on a tab so user can get the error messages

Reply:
This "try-catch" is not for resolving exceptions, just for work around the setOwner()'s error. I cannot set a @gmail.com account when I'm using @coderbunker.com account, because they are under two different domains. But I don't think the owner is a problem so I just need the code to continue running without interruption

Reply:
collect the errors into an array of errors and return it?

and make it explicit, if we know that it's because of not same domain @gmail.com vs @coderbunker.com.

owner is important for personal folders.

Reply:
Add error handling solution:
1. send error emails to the script runner
2. show a dialog in the spreadsheet window to warn about the error
3. record the error messages into a document: [members share -> Coderbunker Systems -> logs -> Coderbunker Scripts Logs - Onboarding](https://docs.google.com/spreadsheets/d/1l05UV303UTPuYOoTDkXkmBB3HqTbRBLitznZxZem02U/edit)
(4. also all errors have already logged  into Google Stackdriver Logging system)